src/components/LandscapeVideoCard/LandscapeVideoCard.jsx

```diff
@@ -22,17 +23,22 @@ function LandscapeVideoCard({
   _id,
   inHistory,
   inLikedVideo,
+  inWatchLater,
 }) {
   const { authState } = useAuth();
   const { historyDispatch } = useHistory();
   const { likeDispatch } = useLike();
+  const { watchLaterDispatch } = useWatchLater();
   const deleteFromList = () => {
     if (!!inHistory) {
       deleteFromHistory(_id, authState.token, historyDispatch);
     }
     if (!!inLikedVideo) {
```

Comment:
Why `!!`? Like what is it's usecase here?

Reply:
It is redundant as it naturally coerces to boolean.